images/macos/helpers/Common.Helpers.psm1

```diff
@@ -166,3 +166,12 @@ function Get-Architecture {
 
     return $arch
 }
+
+Function Test-CommandExists {
```

Comment:
nit: `function` (lower cases) to be consistent with other functions in this helper

Reply:
fixed